src/Modal/Modal.tsx

```diff
@@ -3,12 +3,14 @@ import styled from 'styled-components'
 
 import { fromTheme } from '../utils/styled'
 import { Button } from '../Button'
+import { Heading } from '../Heading'
 
 const Overlay = styled.div`
   width: 100vw;
   height: 100vh;
   background-color: rgba(0,0,0,0.1);
   position: absolute;
+  z-index: 99999999999999999999999;
```

Comment:
Let's add the zIndex map to the theme (then we can override it in glass if needed)